src/Router.cpp

```diff
@@ -17,12 +17,17 @@ void Router::routeRequest(const HTTPRequest &request, HTTPResponse &response)
 		CGIHandler cgiHandler;
 		cgiHandler.setFDsRef(_FDsRef);
 		cgiHandler.setPollFd(_pollFd);
-		// cgiHandler.handleRequest(request, response);
 		cgiHandler.handleRequest(request, response);
-		// std::cout << std::endl << std::endl << std::endl << std::endl;
-		// std::cout << response;
+		return;
 	}
-	else // it is a static request
+	else if (request.getMethod() == "POST")
+	{
+		std::cout << "Router: POST request" << std::endl;
+
+		UploadHandler uploadHandler;
```

Comment:
Does it mean that every POST request is file upload???

Reply:
If not an upload then it'll be an update to a file, probably with CGI, we don't handle it at the moment.
But the code is intended to be like this (and good enought atm I would say)